RecoLocalTracker/SiPixelRecHits/src/SiPixelTemplateReco2D.cc

```diff
@@ -545,19 +563,17 @@ int SiPixelTemplateReco2D::PixelTempReco2D(int id, float cotalpha, float cotbeta
    }
    
    ipass = 0;
-
-//--- Somewhat experimental, keep this commented out:
 //   if(npass == 1) {
-//      // one pass, require that it have iterated
+      // one pass, require that it have iterated

```

Comment:
Is all this commented out code really needed?
If so, please at least restore the ```//``` at the beginning of the line, to improve readability.
(Even better for the readability if you can remove these lines, and the other related ones...)

Reply:
We will remove this commented code